recipes/cytoplay/meta.yaml

```
@@ -0,0 +1,71 @@
+{% set name = "cytoplay" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f5f05be115987b47c8dfed03c4e50a961902cea366d30dc5f6bc379e9d2119eb
+build:
+  skip: true  # [win]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation 
+
+requirements:
+  build:
+    - cmake >=3.14
+    - make 
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - libblas
+    - liblapack
+    - pybind11
+    - freeglut  # [linux]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - xorg-libxi
+    - xorg-libxmu
+    - xorg-libx11  # [linux]
+    - glew
+    - zlib
```

Everything except the CDTs should go into host as they are dependencies this will link against
```suggestion
    - {{ cdt('mesa-libgl-devel') }}  # [linux]
    - {{ cdt('mesa-dri-drivers') }}  # [linux]
```